test/parallel/test-module-children.js

```diff
@@ -0,0 +1,12 @@
+// Flags: --no-deprecation
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const dir = common.fixturesDir + '/GH-7131';
```

Comment:
Mwah, I can change that if you feel strongly but we have a gazillion tests that use simple string concatenation.